http-gateway/web/src/index.js

```diff
@@ -46,7 +46,7 @@ fetch('/web_configuration.json')
         <Provider store={store}>
           <IntlProvider>
             <Auth0Provider
-              domain={domain}
+              domain={authority}
```

Comment:
is it ok? The previous property domain doesn't contain `https://`, now it is part of the property authority.